walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -53,6 +54,12 @@ public Event(String title, TimeRange when, Collection<String> attendees) {
     this.attendees.addAll(attendees);
   }
 
+  public static final Comparator<Event> ORDER_BY_START = new Comparator<Event>() {
```

Comment:
Using lambdas you can simplify this to 
```Comparator<Event> orderByStart = (Event a, Event b) -> a.when.start().compareTo(b.when.start());```
if you want

Reply:
I tried this and it threw an error

Reply:
Oh? What was the error?